private/api/getprofile.php

```diff
@@ -7,5 +7,5 @@
   $steps = escapeshellarg(urldecode($argv[6]));
   $axes = escapeshellarg(urldecode($argv[7]));
 
-  echo json_encode(exec("python private/api/2ptsToProfile.py $path $lat1 $lon1 $lat2 $lon2 $steps $axes 1 2>&1" ));
+  echo json_encode(exec("python3 private/api/2ptsToProfile.py $path $lat1 $lon1 $lat2 $lon2 $steps $axes 1 2>&1" ));

```

Comment:
Do we know the command will be `python3`? Does the Dockerfile's python from `RUN apt-get install -y php7.0-cli php-db php-sqlite3 gdal-bin libgdal-dev python-pip python-gdal` support it?

Reply:
I just updated the Dockerfile so that "python" refers to Python3 and ensured that the GDAL bindings are included in Python3. Also changed "python3" calls back to just "python" to allow any older systems that aren't using Python3 or Docker to still run the PHP scripts.